manager.c

```diff
@@ -297,6 +300,43 @@ static char * espace_newlines(const char * text)
 	return escaped;
 }
 
+#/* */
+static char * unescape_newlines(const char * text)
+{
+	char * unescaped;
+	int i, j;
+	for(j = i = 0; text[i]; ++i)
+	{
+		if(text[i] != '\\' || !(text[i + 1] == 'r' || text[i + 1] == 'n'))
+			j++;
+	}
+	unescaped = ast_malloc(j + 1);
```

Comment:
This backslash-counting before ast_malloc is totally unnecessary. Using strlen() would've been fine and is more readable.